src/schema/nmdc.yaml

```diff
@@ -2781,3 +2808,22 @@ slots:
 
   completion_date:
     range: string
+
+  container_size:
+    range: QuantityValue
+    description: The volume of the container an analyte is stored in or an activity takes place in
+
+  container_type:
+    range: ContainerTypeEnum
+    description: >-
+      A type of container. i.e. test tube, falcon tube, etc.
+
+  volume:
+    range: QuantityValue
+    description: The volume of the solvent/solute being used, not the input.
```

Comment:
Here we are defining a very general schema-wide slot `volume`; we can imagine this being used many places, e.g. volume of a container, of a piece of soil. But the description is very specific. Either do one of:

1. Make a more specific slot such as solvent_volume, and keep the definition
2. Keeping a generic volume, and amending this definition to be more generic, and putting the specific description on slot_usage in SubSamplingProcesss